client/components/editor/teaching-elements/Assessment/feedback/index.vue

```diff
@@ -10,27 +10,22 @@
     </span>
     <transition name="fade">
       <ul v-if="isExpanded">
-        <li
+        <feedback-item
           v-for="(answer, index) in processedAnswers"
-          :key="index">
-          <div>
-            <span class="answer-index">Answer {{ index + 1 }}:</span>
-            {{ answer }}
-          </div>
-          <textarea
-            :ref="'option' + index"
-            :value="feedback ? feedback[index] : ''"
-            :disabled="!isEditing"
-            @change="update(index)"
-            class="form-control">
-          </textarea>
-        </li>
+          :key="index"
+          :answer="answer"
+          :index="index"
+          :feedback="feedback[index]"
+          :isEditing="isEditing"
```

Comment:
Maybe it is just me, but it kinda looks cleaner using `v-bind`:
```html
v-bind="{ index, answer, isEditing }"
:key="index"
:feedback="feedback[index]"
```

What do you think? 🤔